todo-app/src/components/TodoList.js

```diff
@@ -0,0 +1,19 @@
+import React, {Component}  from 'react';
+import TodoItem from './TodoItem';
+import TodoStatus from './TodoStatus';
+import TodoNumber from './TodoNumber';
+
+
+
+export default class TodoList extends Component {
+    
+    render() {
+        const { id, text, status } = this.props.todo;     
+        return (<div>
```

Comment:
Code formatting here is a bit off, makes it harder to read. Especially when we need to write more code and collaborate with other developers.